website/server/libs/user/validation.js

```diff
@@ -29,7 +47,8 @@ function usernameContainsInvalidCharacters (username) {
 export function verifyDisplayName (displayName, res) {
   const issues = [];
   if (displayName.length < 1 || displayName.length > 30) issues.push(res.t('displaynameIssueLength'));
-  if (nameContainsSlur(displayName)) issues.push(res.t('displaynameIssueSlur'));
+  if (stringContainsSlur(displayName)) issues.push(res.t('displaynameIssueSlur'));
```

Comment:
can these two ifs be combined into one? If both are true you would push the same error text twice

Reply:
or maybe we can have different strings for slurs and banned words